src/ui/components/play-toast.ts

```diff
@@ -48,10 +65,11 @@ export class PlayToast extends LitElement {
     this.#timeout = globalThis.setTimeout(() => {
       this.opened = false
       this.#timeout = undefined
-    }, 3000)
+    }, 2000)
   }
 
   protected override render() {
-    return html`<slot></slot>`
+    return html`<play-icon size="24px" icon="checkmark-fill"></play-icon
```

Comment:
:no_bell: I think this icon should be outside the toast so we can use it for things besides confirmation but not a big deal.

Reply:
Agreed. Making the icon a fixed part of the toast makes the component overall less versatile.

I don't think we'll use toasts for error messages, given it's short display duration, but it's certainly possible for messages of more neutral intent that doesn't warrant the icon.